src/main/scala/com/gu/salesforce/messageHandler/MessageHandlerLambda.scala

```diff
@@ -24,21 +22,6 @@ trait MessageHandler extends Logging {
 
   val queueName = s"salesforce-outbound-messages-${Config.stage}"
 
-  def parseMessage(requestBody: String) = {
-    val is = new ByteArrayInputStream(requestBody.getBytes)
-    val messageFactory = MessageFactory.newInstance()
-    val soapMessage = messageFactory.createMessage(null, is)
-    val body = soapMessage.getSOAPBody
-    val jc = JAXBContext.newInstance(classOf[Notifications])
-    val unmarshaller = jc.createUnmarshaller()
-    val je = unmarshaller.unmarshal(body.extractContentAsDocument(), classOf[Notifications])
-    je.getValue()
-  }
-
-  case class QueueMessage(contactId: String)
-
-  implicit val messageFormat = Json.format[QueueMessage]
```

Comment:
Moved this to it's own file, just because I get a bit itchy when case classes are members of traits or class instances when they don't need to be - we shouldn't have to create an instance of `MessageHandler` just to create a `QueueMessage`!